src/simmate/database/base_data_types/staged_calculation.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import logging
+from pathlib import Path
+
+from simmate.database.base_data_types import Calculation, Structure, table_column
+
+
+class StagedCalculation(Structure, Calculation):
```

Comment:
I don't think `StagedCalculation` should be a standalone table, but a mixin. 

For total beginners that run a staged workflow, there's a big difference between giving a table that says... "here is where you can go find the results" vs. "here are the final results, and there are some extra columns that say where we got them from".

The staged relaxation and population analysis tables that I currently have in `main` are more of the latter, while your PR is pushing the former. Your approach is pretty common in other software (bc it saves on disk space), but it's just a pain for new users. So my push back is more of a personal preference + design philosophy thing

So just as a heads up, once you're ready for review+edits, I'll be modifying some of your tables to clean things up